### PR TITLE
🎉 New Source: Clazar

### PR DESCRIPTION
**Pull request by prashant-mittal9**
## What
* New source: Clazar. [docs](https://developers.clazar.io/)

## How
* Using connector builder and low code template provided.

## Review guide
1. `manifest.yaml`

## User Impact
* Users will get new source!!

## Can this PR be safely reverted and rolled back?
- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Review by marcosmarxm — COMMENTED** [review]
@prashant-mittal9 implementation looks good. Need to add credentials and run tests.

**Comment by prashant-mittal9** [comment]
> @prashant-mittal9 implementation looks good. Need to add credentials and run tests.

will provide over slack DM

File: airbyte-integrations/connectors/source-clazar/integration_tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-clazar/integration_tests/__init__.py:1]: If there are no actual integration tests, would you please remove the python files in this directory?

File: airbyte-integrations/connectors/source-clazar/integration_tests/acceptance.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-clazar/integration_tests/acceptance.py:1]: I.e.  this — I believe you can safely delete the file with no impact to actually running tests.

File: airbyte-integrations/connectors/source-clazar/integration_tests/sample_config.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-clazar/integration_tests/sample_config.json:1]: That should be removed — CATs would fetch the secrets file from the secrets manager when configured correctly. On a very low usage (when we just get started) it's okay to not have this here at all.

File: airbyte-integrations/connectors/source-clazar/pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/clazar"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+packages = [ { include = "source_clazar" }, {include = "main.py" } ]
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-clazar/pyproject.toml:19]: Please switch this to "< 3.0" and then run `poetry lock` again. That's a mistake on our side — the recent CDK is 2.0+ already. 

```suggestion
airbyte-cdk = "< 3.0.0"
```

File: airbyte-integrations/connectors/source-clazar/poetry.lock [hunk 5]
```diff
@@ -0,0 +1,1309 @@
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+
+[[package]]
+name = "airbyte-cdk"
+version = "0.90.0"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-clazar/poetry.lock:5]: After running `poetry lock` on fixed CDK version this should go to 2.0+.

**Review by natikgadzhi — COMMENTED** [review]
I'd love to have a bit of cleanup here.

If Clazar source is a fully low-code source with absolutely no Python custom components, and if you have not added any integration tests, than do this: 
1. Move `manifest.yaml` to the root level, it should sit alongside `metadata.yaml` file
2. Delete everything else. This PR should then include `manifest.yaml`, `metadata.yaml`, the icon file, and the readme file.
3. Once @bnchrch merges his "slim connector" build pipeline, we will be able to merge and publish this. 

All in all, I would LOVE to leave just the yamls + readme here, and then I can get this merged later this week I think.

File: airbyte-integrations/connectors/source-clazar/metadata.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
+data:
+  allowedHosts:
+    hosts:
+      - api.clazar.io
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: false
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-clazar/metadata.yaml:9]: @prashant-mittal9 you probably want to set this to true if you want this connector to work on Airbyte Cloud.

File: airbyte-integrations/connectors/source-clazar/source_clazar/manifest.yaml [hunk 7]
```diff
@@ -0,0 +1,4497 @@
+version: 0.90.0
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-clazar/source_clazar/manifest.yaml:1]: Set this to whatever CDK version will install with poetry lock — likely 2.0+

**Review by natikgadzhi — APPROVED** [review]
Alright, LGTM.

**Pull request merged**